src/components/_global/BalCard/BalCard.vue

```diff
@@ -23,14 +24,15 @@ export default defineComponent({
 
   props: {
     title: { type: String, default: '' },
-    titleTag: { type: String, default: 'h5' },
+    titleTag: { type: String, default: 'h4' },
     square: { type: Boolean, default: false },
     noPad: { type: Boolean, default: false },
     noContentPad: { type: Boolean, default: false },
     noBorder: { type: Boolean, default: false },
     darkBgColor: { type: String, default: '850' },
-    maxHeight: { type: Boolean, default: false },
-    greedyContent: { type: Boolean, default: false },
+    imgSrc: { type: String, default: '' },
+    hFull: { type: Boolean, default: false },
+    growContent: { type: Boolean, default: false },
```

Comment:
I think it's better to be as explicit as possible when naming props for these low level components. For example, unless you implemented it 'greedyContent' is quite difficult to understand at first glance.